test_scripts/API/SDL_Passenger_Mode/001_OnDriverDistraction_with_lockScreenDismissalEnabled_for_all_states.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0119-SDL-passenger-mode.md
+--
+-- Description:
+-- In case:
+-- 1) By policy OnDriverDistraction allowed for (FULL, LIMITED, BACKGROUND, NONE) HMILevel
+-- 2) App registered and activated (HMI level FULL)
+-- 3)  HMI sends "lockScreenDismissalEnabled" item (boolean) (with all mandatory fields) as a parameter
+--     of OnDriverDistraction notification
+-- Note: Covers all hmi levels
+-- SDL does:
+-- 1) Send OnDriverDistraction notification to mobile with "lockScreenDismissalEnabled" parameter and all mandatories
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SDL_Passenger_Mode/commonPassengerMode')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+
+runner.Title("Test")
+runner.Step("App registration HMI level NONE", common.registerApp)
+for _, v in pairs(common.OnDDValue) do
+	runner.Step("OnDriverDistraction with state " .. v .. " with lockScreenDismissalEnabled true",
+	common.onDriverDistraction, { v, true })
+end
+
+for _, k in pairs(common.OnDDValue) do
+	runner.Step("OnDriverDistraction with state " .. k .. " with lockScreenDismissalEnabled false",
+	common.onDriverDistraction, { k, false })
+end
```

Comment:
@HSavynetska There is no need to create 2 loops - one after another. Just add outer loop with (true, false)